### PR TITLE
add tippy style

### PR DESCRIPTION
**Pull request by natth42**

### CODE REVIEW

File: scss/themes/_tippy.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@import './tippy/base';
+@import './tippy/themes/gorila-dark';
```
Comment (oliveira-gust) [inline scss/themes/_tippy.scss:2]: You could add gray and white styles too

File: scss/themes/tippy/themes/_gorila-gray.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
+@import '../mixins';
+@import '../variables';
+
+$gorilaGrayBg: $base-background;
+$gorilaFontColor: $extra-dark-gray;
```
Comment (oliveira-gust) [inline scss/themes/tippy/themes/_gorila-gray.scss:5]: Please remove these and update the files to use colors from our color files

Comment (natth42) [reply scss/themes/tippy/themes/_gorila-gray.scss:5]: Do you mean use the variables $base-background and $extra-dark-gray directly?

Comment (oliveira-gust) [reply scss/themes/tippy/themes/_gorila-gray.scss:5]: And if there's any color that is here but not in the colors file, add it there too

**Pull request merged**